pages/productListing.html

```diff
@@ -0,0 +1,310 @@
+        <div class="quantity-slider">
+            <div class="title semibold">Price Range</div>
+            <div class="slider-range">
+              <span class="slider-range-min">50</span>
+              <span class="slider-range-mid">150</span>
+              <span class="slider-range-max">200</span>
+            </div>
+            <input type="range" min="1" max="5000" class="slider" />
+          </div>
+          <div class="list-container">
+            <div class="title semibold">Category</div>
+            <ol class="list list-none">
+                <li class="list-items"><label class="is-light form-checkbox is-2">
+        <input type="checkbox">
+        Stocks
+      </label></li>
+                <li class="list-items"><label class="is-light form-checkbox is-2">
+        <input type="checkbox">
+        Forex
+      </label></li>
+                <li class="list-items"><label class="is-light form-checkbox is-2">
+        <input type="checkbox">
+        Crypto
+      </label></li>
+                <li class="list-items"><label class="is-light form-checkbox is-2">
+        <input type="checkbox">
+        Charting
+      </label></li>
+                <li class="list-items"><label class="is-light form-checkbox is-2">
+        <input type="checkbox">
+        Biography
+      </label></li>
+            </ul>
+        </div><div class="list-container">
+            <div class="title semibold">Rating</div>
+            <ol class="list list-none">
+                <li class="list-items"><label class="form-radio-label">
+                    <input type="radio" name="opinion" checked />
+                    <i></i>
+                    <span class="form-radio-button-text is-2 is-medium">4 Star & above</span>
+                  </label></li>
+                <li class="list-items"><label class="form-radio-label">
+                    <input type="radio" name="opinion" checked />
+                    <i></i>
+                    <span class="form-radio-button-text is-2 is-medium">3 Star & above</span>
+                  </label></li>
+                <li class="list-items"><label class="form-radio-label">
+                    <input type="radio" name="opinion" checked />
+                    <i></i>
+                    <span class="form-radio-button-text is-2 is-medium">2 Star & above</span>
+                  </label></li>
+                <li class="list-items"><label class="form-radio-label">
+                    <input type="radio" name="opinion" checked />
+                    <i></i>
+                    <span class="form-radio-button-text is-2 is-medium">1 Star & above</span>
+                  </label></li>
+            </ul>
+        </div>
+        <div class="list-container">
+            <div class="title semibold">Sort By</div>
+            <ol class="list list-none">
+                <li class="list-items"><label class="form-radio-label">
+                    <input type="radio" name="opinion" checked />
```

Comment:
You can change the value of the name attribute here. "Sort by" and "Rating" should have different radio names.

Reply:
Yes I will see to that